admin_manual/appliance/enterprise-trial.rst

```diff
@@ -0,0 +1,25 @@
+The appliance has the community edition of ownCloud but can be easily upgraded to the enterprise edition. This upgrade gives you access to a Free-30-Day-Trial of the enterprise edition and all it's features. All you need is an email address to get started. Here are the necessary steps:
```

Comment:
s/has the community/contains the community/g. Alternatively, you could also say "is built on" instead of "contains".